### ISSUE TITLE
Please add Changelog

### ISSUE THREAD

**Issue by fabioimpe** [twobin/react-lazyload#221]
Title says all, please add the changelog when a new release is available

**Comment by ameerthehacker** [comment]
Sure @twobin that makes sense

**Comment by tstirrat15** [comment]
Or put an entry in [releases](https://github.com/twobin/react-lazyload/releases). I've seen a lot of projects do it that way.

**Comment by ameerthehacker** [comment]
Yes @Fandekasp that is onething that we have been thinking about

**Comment by joris-** [comment]
I can help you to setup a tool like [release-it](https://github.com/release-it/release-it)

**Comment by ameerthehacker** [comment]
@joris- please goahead

**Comment by olso** [comment]
@jorisre i guess u werent tagged properly

**Comment by jorisre** [comment]
@ameerthehacker what do you think about https://github.com/semantic-release/semantic-release ?

**Comment by jorisre** [comment]
@ameerthehacker I don't have sufficient permissions to setup semantic-release _(need npm credentials, etc..)_ 
I think, it's better if you set it up.

➡️  https://github.com/semantic-release/semantic-release/blob/master/docs/usage/getting-started.md#getting-started

**Comment by monsieurnebo** [comment]
Hi,

Please consider adding a changelog to react-lazyload. 

This is pretty hard to update this package without knowing what is gonna (break) change.
